app/assets/stylesheets/components-vue/_card.scss

```diff
@@ -9,13 +9,13 @@
     padding: 0;
     text-align: left;
     text-decoration: none;
-    height: rem-calc(190);
+    min-height: rem-calc(400);
```

Comment:
The issue with the text is that the overlay element it's within is absolutely positioned (I had to make it so because it would cause the text to expand and contract with the background image when hovered over) so the height of the card is independent of the height of the text. The best I can do with this approach is to make them have the same minimum min-height. 

I could make the image a background image of the card but that also means I'll have to rework the overlay (the dark gradient and the zooming-in effect). Not sure how to as this current setup is what I arrived at after quite a lot of experimentation.

Reply:
Thanks, I got it working